src/scripts/main.js

```diff
@@ -1,3 +1,90 @@
 'use strict';
 
-// write code here
+const tHead = document.querySelector('thead');
+const tBody = document.querySelector('tbody');
+const rows = [...tBody.children];
+let reverse = false;
+
+function handleEvent(e) {
+  let newList = [...rows];
+
+  if (reverse) {
+    reverse = false;
+
+    switch (e.target.textContent) {
+      case 'Name':
+        newList = [...rows].sort(
+          (a, b) =>
+            b.children[0].textContent.localeCompare(a.children[0].textContent)
+        );
+        break;
+
+      case 'Position':
+        newList = [...rows].sort(
+          (a, b) =>
+            b.children[1].textContent.localeCompare(a.children[1].textContent)
```

Comment:
In both cases you do almost the same. Think about combining similar cases. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#taking_advantage_of_fall-through